controllers/api/api_event_controller.py

```diff
@@ -196,7 +197,7 @@ def _track_call(self, *args, **kw):
         self._track_call_defer('event/list', self.year)
 
     def _render(self, year=None):
-        if self.year < 1992 or self.year > datetime.now().year + 1:
+        if self.year < tba_config.MIN_YEAR or self.year > datetime.now().year + 1:
```

Comment:
I'm not going to modify this bit to use `VALID_YEARS` - since it doesn't look at our constant year range, it looks at the current date (is this correct behavior?)